app/src/main/java/com/github/epfl/meili/home/GoogleSignInActivity.kt

```diff
@@ -28,7 +28,7 @@ class GoogleSignInActivity : AppCompatActivity() {
         updateUI()
     }
 
-    fun onGoogleButtonClick(view: View) {
+    fun onGoogleButtonClick(@Suppress("UNUSED_PARAMETER") view: View) {
```

Comment:
out of curiosity what does @Supress("UNUSED_PARAMETER") do?

Reply:
Without this annotation the build generates an unused parameter warning. Since we can't remove this from the function declaration as it's a onClick listener, this is the only way to not have this warning